### PR TITLE
Address feedback from #60

### PR DESCRIPTION
**Pull request by juancampa**
Addressing the feedback left in #60 since I accidentally merged it.

 - Made the visual hierarchy more prominent by
   - Adding a margin to paragraphs so they are indented
   - Added a background to headings.
   - Made h1 headers bigger
 - Rephrased the changes made to the `state` page.
 - Removed the background to the code blocks to reduce the number of gray things.

### CODE REVIEW

**Review by pmillspaugh — COMMENTED** [review]
Nice, getting more Membrane-y

Haha, this font is small! I'll zoom to 150% when reading these on my laptop, which is ok—curious what others think.

The notes/tips/etc. have an extra indentation (from nested padding) now:

Screenshot 2024-11-26 at 4 35 02 PM

There's also extra indentation on code block tooltips:

Screenshot 2024-11-26 at 4 37 25 PM

Since the code blocks are white now...should we add the box shadow that images and videos have? ❑

Screenshot 2024-11-26 at 4 39 18 PM

**Comment by iamseeley** [comment]
Screenshot 2024-11-26 at 4 58 25 PM

Here is what a 14px font for p, removed left-padding on p, and removed top margin on h1 looks like.

File: src/styles/base.css [hunk 1]
```diff
@@ -23,18 +25,38 @@ html {
...
+:root[data-theme="light"] h2,
+:root[data-theme="light"] h3,
+:root[data-theme="light"] h4,
+:root[data-theme="light"] h5,
+:root[data-theme="light"] h6 {
+  background-color: var(--sl-color-gray-6);
+}
+
+.sl-markdown-content p {
+  padding-left: 2ch;
```
Comment (iamseeley) [inline src/styles/base.css:59]:
```suggestion
.sl-markdown-content p {
  font-size: 14px;
}

.sl-markdown-content h1 {
  margin-top: 0;
}
```
Going to leave this suggestion here if you all like the larger p font without the left padding and the margin-top of h1 removed

Comment (iamseeley) [reply src/styles/base.css]: I see! Yeah, makes sense. 13px looks good to me 👍

**Comment by juancampa** [comment]
> Since the code blocks are white now...should we add the box shadow that images and videos have? ❑

I'm leaving it transparent for now to reduce visual noise. I think it's easier to parse that way. We can revisit later for sure.

Thank you guys!

**Pull request merged**